internal/gateway/listbgpneighbours.go

```diff
@@ -0,0 +1,51 @@
+package gateway
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) ListBgpNeighbours() *cobra.Command {
+	var gatewayId string
+
+	// createMetalGatewayCmd represents the createMetalGateway command
+	createMetalGatewayCmd := &cobra.Command{
+		Use:   `list-bgp-dynamic-neighbours`,
+		Short: "Lists BGP Dynamic Neighbours for Metal Gateway",
+		Long:  "Lists the BGP Dynamic Neighbour for the metal gateway with the specified gateway ID",
+		Example: `# Lists BGP Dynamic Neighbour for the specified metal gateway ID
+
+	$ metal gateways list-bgp-dynamic-neighbour --id "9c56fa1d-ec05-470b-a938-0e5dd6a1540c"
+`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			n, _, err := c.VrfService.
+				GetBgpDynamicNeighbors(context.Background(), gatewayId).
+				Include(c.Servicer.Includes([]string{"created_by"})).
+				Exclude(c.Servicer.Excludes([]string{})).
+				Execute()
+			if err != nil {
+				return fmt.Errorf("Could not list BGP Dynamic Neighbours for Metal Gateway [%s]: %w", gatewayId, err)
+			}
+
+			data := make([][]string, len(n.GetBgpDynamicNeighbors()))
+			for _, neighbour := range n.GetBgpDynamicNeighbors() {
+				data[0] = []string{neighbour.GetId(), neighbour.GetBgpNeighborRange(),
+					strconv.Itoa(int(neighbour.GetBgpNeighborAsn())), string(neighbour.GetState()), neighbour.GetCreatedAt().String()}
+			}
+
+			header := []string{"ID", "Range", "ASN", "State", "Created"}
+			return c.Out.Output(n, header, &data)
+		},
+	}
+
+	createMetalGatewayCmd.Flags().StringVar(&gatewayId, "gateway-id", "", "")
```

Comment:
This should be `id` to match the docs

```suggestion
	createMetalGatewayCmd.Flags().StringVar(&gatewayId, "id", "", "")
```

Reply:
fixed